rclcpp_action/test/test_traits.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <test_msgs/action/fibonacci.hpp>
+
+using rosidl_generator_traits::is_message;
+using rosidl_generator_traits::is_service;
+using rosidl_generator_traits::is_action;
+using rosidl_generator_traits::is_action_goal;
+using rosidl_generator_traits::is_action_result;
+using rosidl_generator_traits::is_action_feedback;
+
+TEST(TestActionTraits, is_action) {
+  using Fibonacci = test_msgs::action::Fibonacci;
+
+  // Top level definition is an action
+  ASSERT_FALSE(is_message<Fibonacci>());
```

Comment:
nit: using `EXPECT_*` instead, we'd see multiple failures instead of just the first to fail.